index.ts

```diff
@@ -27,4 +45,19 @@ const getImage = async (event: EventDto): Promise<any> => {
 	}
 };
 
+const getFileS3 = async (bucketName: string, key: string): Promise<Buffer> => {
+	const data: any = await s3.getObject({Bucket: bucketName, Key: key}).promise();
+	return data?.Body as Buffer;
```

Comment:
not safe type force
data?.Body with “?.” mean it can be undefined
as Buffer: mean you force cast to not undefined

should using not null/undefined check so that typescript can infer the variable is not null/undefined